tests/test_layers.py

```diff
@@ -1049,6 +1049,25 @@ def model_func(wrapped_import, inputs: torch.Tensor):
   verify_torch_and_convert_to_returnn(model_func, inputs=x)
 
 
+def test_unsqueeze3():
+  from pytorch_to_returnn.naming import Naming
+  n_batch, n_time = 3, 7
+
+  def model_func(wrapped_import, inputs: torch.Tensor):
+    out = inputs.unsqueeze(1)
+    naming = Naming.get_instance()
+    if naming.import_params_from_torch_namespace:
+      returnn_data = naming.tensors[out].returnn_data
+      assert returnn_data.time_dim_axis == 2
+      assert returnn_data.feature_dim_axis == 1
```

Comment:
This should not matter here, so you should not test this. This can be arbitrary.

Make a test case for what sth which actually fails.